.github/workflows/regression-tests.yml

```diff
@@ -102,10 +102,10 @@ jobs:
         command: pip install -U pip  # upgrade pip after installing python
     - name: Set up Anaconda ${{matrix.python-version}} (Windows)
       if: startsWith(matrix.os, 'windows')
-      uses: conda-incubator/setup-miniconda@505e6394dae86d6a5c7fbb6e3fb8938e3e863830  # v3.1.1
+      uses: conda-incubator/setup-miniconda@835234971496cad1653abb28a638a281cf32541f  # v3.2.0
       with:
-        auto-update-conda: true
```

Comment:
Saves time and prevents issues with being unable to solve the environment. We don't need everything the absolute latest.